packages/forms-web-app/__tests__/unit/middleware/check-appeal-exists.test.js

```diff
@@ -23,51 +23,14 @@ describe('middleware/check-appeal-exists', () => {
 		};
 	});
 
-	it('should call next() for the `/before-you-start/local-planning-department` page', () => {
-		req.originalUrl = '/before-you-start/local-planning-department';
-		checkAppealExists(req, res, next);
-		expect(next).toBeCalled();
-		expect(res.redirect).not.toBeCalled();
-	});
-
-	it('should call next() for the `/before-you-start/type-of-planning-application` page', () => {
-		req.originalUrl = '/before-you-start/type-of-planning-application';
-		checkAppealExists(req, res, next);
-		expect(next).toBeCalled();
-		expect(res.redirect).not.toBeCalled();
-	});
-
-	it('should call next() for the `/before-you-start/use-a-different-service` page', () => {
-		req.originalUrl = '/before-you-start/use-a-different-service';
-		checkAppealExists(req, res, next);
-		expect(next).toBeCalled();
-		expect(res.redirect).not.toBeCalled();
-	});
-
-	it('should call next() for the `/appellant-submission/submission-information/6e1195ad-176d-4ca3-a944-525218780a7e` page', () => {
```

Comment:
slightly worried about these two submission-information routes but as far as I can tell they are called when submitting an appeal so the appeal will be in session